src/Symfony/Component/Form/Extension/Csrf/EventListener/CsrfValidationError.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Extension\Csrf\EventListener;
```

Comment:
Maybe move the class one level up and rename it to `ValidationError` so that the FQCN reads `Symfony\Component\Form\Extension\Csrf\ValidationError`?